backend/app/modules/access/schemas.py

```diff
@@ -63,6 +63,30 @@ class ConfirmForgotPasswordRequest(BaseModel):
 class ConfirmForgotPasswordResponse(BaseModel):
     message: str
 
+class InviteResponse(BaseModel):
```

Comment:
It appears the frontend isn't expecting a response for a successful invitation. So I think we can delete this data schema.